quarkchain/cluster/tests/test_shard_db_operator.py

```diff
@@ -17,3 +47,96 @@ def test_get_minor_block_by_hash(self):
 
         self.assertEqual(db.get_minor_block_header_by_hash(block_hash), block.header)
         self.assertIsNone(db.get_minor_block_header_by_hash(b""))
+
+    def test_get_transaction_by_address(self):
+        id1 = Identity.create_random_identity()
+        acc0 = Address.create_random_account(full_shard_key=0)
```

Comment:
rename `acc0` to miner_addr so readers will not confuse this address with others